lib/helpers.js

```diff
@@ -124,18 +124,19 @@ async function runQuery(functionName, queryString, queryParams, pool, clientPara
   }
 }
 
-function getAlertTypeDisplayName(alertType) {
+function getAlertTypeDisplayName(alertType, language, t) {
```

Comment:
Are there already tests for this function? If so, could you please extend them to test the parts that you changed?

Reply:
It looks like there aren't yet tests for this function. Can you please add some? Thanks!